### DISCUSSION TITLE
Oxalis 5.X and Tomcat 10.X

### DISCUSSION THREAD

**Discussion by RichardVanMaaren** [OxalisCommunity/oxalis#544] [General]
Did see another topic on EOL of java 8 and oxalis. Question : Does the latest Oxalis release work on tomcat 10.X which is also a higher java version the Java 8 ? Somebody any experience with this ? (javax/jakarta)

Regards,

Richard

**Comment by aaron-kumar** [comment]
Current Oxalis releases are being tested with Tomcat 9.x . 
We will recommend right container version once we will upgrade to Java 17

**Comment by RichardVanMaaren** [comment]
But Tomcat will still be supported in future ? I noticed Dockers seems the advised platform now ?

Regards,

Richard

**Comment by aaron-kumar** [comment]
There is no planned deviation or preference of one container over other, however it is found after recent discussions with in Oxalis community that Oxalis users are preferring Docker more over other. Our main aim is to provide minimum level support to most of container type including tomcat, docker etc.
